packages/react-reconciler/src/ReactFiberHostContext.js

```diff
@@ -150,13 +131,13 @@ function popHostContext(fiber: Fiber): void {
       pop(hostTransitionProviderCursor, fiber);
 
       // When popping the transition provider, we reset the context value back
-      // to `null`. We can do this because you're not allowd to nest forms. If
+      // to `NotPendingTransition`. We can do this because you're not allowd to nest forms. If
```

Comment:
might as well while we're here
```suggestion
      // to `NotPendingTransition`. We can do this because you're not allowed to nest forms. If
```